cmd/ak/cmd/triggers/create.go

```diff
@@ -52,41 +53,33 @@ var createCmd = common.StandardCommand(&cobra.Command{
 			eid = e.ID()
 		}
 
-		// Mode 1: connection is required, event and/or filter
-		// and/or data are required, and schedule is not allowed.
```

Comment:
I agree that the code you wrote instead of this is more generic, so it's better, but it also hides the specific needs of each mode: it's not clear what are the required and optional flags for each mode. WDYT?

Reply:
can you suggest a comment?